fluid-python/FluidSolver.py

```diff
@@ -100,18 +113,19 @@
 if interface.is_read_data_available():
     crossSectionLength = interface.read_block_scalar_data(crossSectionLengthID, vertexIDs)
 
-crossSectionLength_n = np.copy(crossSectionLength)
-velocity_n = config.velocity_in(0) * crossSectionLength_n[0] * np.ones(N+1) / crossSectionLength_n  # initialize such that mass conservation is fulfilled
+crossSectionLength_old = np.copy(crossSectionLength)
+velocity_old = velocity_in(0) * crossSectionLength_old[0] * np.ones(N+1) / crossSectionLength_old  # initialize such that mass conservation is fulfilled
 
-print(crossSectionLength_n)
+print(crossSectionLength_old)
 
+time_it = 0
 while interface.is_coupling_ongoing():
     # When an implicit coupling scheme is used, checkpointing is required
     if interface.is_action_required(action_write_iteration_checkpoint()):
         interface.mark_action_fulfilled(action_write_iteration_checkpoint())
 
-    velocity, pressure, success = perform_partitioned_implicit_euler_step(velocity_n, pressure_n, crossSectionLength_n,
-                                                                          crossSectionLength, dx, precice_dt, config.velocity_in(t + precice_dt), custom_coupling=False)
+    velocity, pressure, success = perform_partitioned_implicit_euler_step(velocity_old, pressure_old, crossSectionLength_old,
+                                                                          crossSectionLength, dx, precice_dt, velocity_in(t + precice_dt), custom_coupling=True)
```

Comment:
Why the False/True change?

Reply:
I changed that to have the same we do in the cpp-solver. In any case, it didn't notice any difference in the results (at least not an obvious one).